pkg/util/kubernetes/autoscalers/processor.go

```diff
@@ -56,9 +60,15 @@ type queryResponse struct {
 // NewProcessor returns a new Processor
 func NewProcessor(datadogCl datadogclientcomp.Component) *Processor {
 	externalMaxAge := math.Max(pkgconfigsetup.Datadog().GetFloat64("external_metrics_provider.max_age"), 3*pkgconfigsetup.Datadog().GetFloat64("external_metrics_provider.rollup"))
+	parallelQueries := pkgconfigsetup.Datadog().GetInt("external_metrics_provider.max_parallel_queries")
```

Comment:
Not for this PR, but it would be great if instead accessing the config struct via `pkgconfigsetup` we would pass the config component 😄